.github/ISSUE_TEMPLATE.md

```diff
@@ -3,13 +3,13 @@
 **Install Method:** How did you install the CLI? (e.g. pip, interactive script, apt-get, Docker, MSI, nightly)  
 Answer here:
 
-**CLI Version:** What version of the CLI and modules are installed? (Available with `az --version`)  
+**CLI Version:** What version of the CLI and modules are installed? (The full output of `az --version`)  
 Answer here:
 
 **OS Version:** What OS and version are you using?  
 Answer here:
 
-**Shell Type:** What shell are you using? (e.g. bash, cmd.exe, PowerShell)  
+**Shell Type:** What shell are you using? (e.g. bash, cmd.exe, PowerShell, Bash on Windows)  
```

Comment:
I don't think we should put Powershell in this list since it's not one we support. This makes it look like we do.